src/pages/style-guide.mdx

```diff
@@ -0,0 +1,162 @@
+import CondaCSVG from '@site/static/img/conda_c.svg';
+import CondaLogoSVG from '@site/static/img/conda_logo.svg';
+
+
+# Style Guide
+
+This page serves as a style guide for conda related projects.
+It contains important logos and fonts to use when creating your own conda
+branded content or websites.
+
+---
+
+## Typography
+
+Below are various typography elements. The header font for [conda.org](https://conda.org) is 
+[Nunito](https://fonts.google.com/specimen/Nunito) and the body font is 
+[Open Sans](https://fonts.google.com/specimen/Open+Sans).
+
+# Header one (2rem)
```

Comment:
I think Docusaurus only allows one `# header` per document, so this one is stripped out from the sidebar (weirdly). Maybe we can hardcode it `<h1>...</h1>`?

Reply:
Oh weird. Good catch. That should be easy enough to do.